### PR TITLE
Add AEM 6.5 support

### LINKED ISSUE #66: Add AEM 6.5 support

### PR DESCRIPTION
**Pull request by bdhoine**
- Add AEM 6.5 support
- Cleanup TranslationConfig mock injection
- Correctly transpile clientlibs

Resolves #66, #61 , #65, #42

**Checklist before requesting a review**

- [x] Contribution guidelines read in [CONTRIBUTING.md](CONTRIBUTING.md)
- [x] I have performed a self-review of my code
- [x] Tests are added where appropriate
- [x] The resolution of the issue is mentioned in the [CHANGELOG.md](CHANGELOG.md)

### CODE REVIEW

**Comment by bdhoine** [comment]
@kwin this is the pr that contains the fix for the compilation and adds AEM 6.5 support, I tested on a AEM 6.5.17 instance and a AEMaaCS SDK (2024.10)

File: ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/clientlibs/dictionary/js/publish.js [hunk 1]
```diff
@@ -49,7 +49,7 @@
         }).done(function () {
             var api = $(collection).adaptTo("foundation-collection");
             if (api && "reload" in api) {
-                api.reload();
+                // api.reload();
```
Comment (bdhoine) [inline ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/clientlibs/dictionary/js/publish.js:52]: I overlooked this one, on 6.5.17 it's throwing an exception, currently investigating why

File: ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/clientlibs/dictionary/js/action.js [hunk 2]
```diff
@@ -42,31 +42,32 @@
...
             }
         }).always(function () {
             ui.clearWait();
         }).done(function () {
-            var api = $(collection).adaptTo("foundation-collection");
+            let api = $(collection).adaptTo("foundation-collection");
             if (api && "reload" in api) {
-                api.reload();
+                // TODO: Throwing exception in 6.5
+                // api.reload();
```
Comment (bdhoine) [inline ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/clientlibs/dictionary/js/action.js:53]: needs to be fixed or removed

**Review by royteeuwen — COMMENTED** [review]
if you fix your own remarks, it should be fine :D

**Review by royteeuwen — APPROVED** [review]

**Pull request merged**